recipes/procps/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "procps" %}
+{% set version = "3.3.12" %}
+{% set sha256 = "b1036c109f271c7c50325b11a748236f8e58aa1dbafefb30c995ff1d05b4a1a8" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://gitlab.com/procps-ng/procps/repository/v{{ version }}/archive.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    skip: True  # [win or osx]
+    
+requirements:
+    build:
+        - toolchain
+        - automake
+        - autoconf
+        - pkg-config
+        - libtool
+        - gettext
+        - ncurses ==5.9
+    run:
+        - ncurses ==5.9
+
+test:
+    commands:
+        # Test if ps is working
+        - "ps --version"
+        - "ps"
```

Comment:
Should we check some of the other programs it includes? Also might want to add the full path here just to be sure (e.g. `"${PREFIX}/bin/ps"`).